src/freenet/clients/fcp/ClientRequest.java

```diff
@@ -27,6 +27,9 @@
  */
 public abstract class ClientRequest implements Serializable {
 
+    /**
+     * ATTENTION: When incrementing this, please skip version 2. Version 2 had already temporarily
+     * been used by a development branch. */
```

Comment:
It's true that the purge-db4o snapshots / preleases have been out a long time - my question is why blacklist UID 2 when its usage was never even merged?